src/ompl/base/OptimizationObjective.h

```diff
@@ -97,6 +125,12 @@ namespace ompl
             /** \brief Get the cost that corresponds to combining the costs \e c1 and \e c2. Default implementation defines this combination as an addition. */
             virtual Cost combineCosts(Cost c1, Cost c2) const;
 
+            /** \brief Combine three costs as (c1 + c2) + c3. By default uses the two-argument version of combineCosts */
+            virtual Cost combineCosts(Cost c1, Cost c2, Cost c3) const;
+
+            /** \brief Combine four costs as ((c1 + c2) + c3) + c4. By default uses the two-argument version of combineCosts */
+            virtual Cost combineCosts(Cost c1, Cost c2, Cost c3, Cost c4) const;
+
```

Comment:
Are there use cases for non-default behavior for the three- and four-argument version of CombineCosts? If not, then I'd rather not add these.